Utils/Dataflow/pyDKB/common/types.py

```diff
@@ -0,0 +1,9 @@
+"""
+pyDKB.common.types
+
+Definitions of types used across all the library moduless.
```

Comment:
Double 's' in 'modules'.

Reply:
Fixed: 678604ae903c5e1dd91409629ea8e08edbbc22ec.